### DISCUSSION TITLE
filter plots

### DISCUSSION THREAD

**Discussion by Bastiebb** [mboquien/cigale#92] [General]
I know there is a way for CIGALE to plot the transmission curves of different filters, but is there a way for CIGALE to present a plot with multiple plots?

**Comment by mboquien** [comment]
At this moment the `pcigale-filters plot` onlt generates one plot per filter. However if you want to plot several filters, it is easy to tap into the database with a few lines.
```python
import matplotlib.pyplot as plt
import numpy as np
from pcigale.data import SimpleDatabase as Database

db = Database("filters")
F555W = db.get(name="hst.wfc3.F555W")
plt.plot(F555W.wl, F555W.tr / np.max(F555W.tr))
```
etc.

**Reply by Bastiebb** [reply]
Thank you, How would this work if I am using both filters that have a #photon unit and an #energy unit?
